pkg/operator2/operator.go

```diff
@@ -200,35 +208,36 @@ func (c *authOperator) handleSync(operatorConfig *operatorv1.Authentication) err
 	if err != nil {
 		return err
 	}
+	resourceVersions = append(resourceVersions, sessionSecret.GetResourceVersion())
 
 	consoleConfig := c.handleConsoleConfig()
+	resourceVersions = append(resourceVersions, consoleConfig.GetResourceVersion())
 
 	oauthConfig, expectedCLIconfig, syncData, err := c.handleOAuthConfig(operatorConfig, route, service, consoleConfig)
 	if err != nil {
 		return err
 	}
+	resourceVersions = append(resourceVersions, oauthConfig.GetResourceVersion())
+
+	configResourceVersions, err := c.handleConfigSync(syncData)
+	if err != nil {
+		return err
+	}
+	resourceVersions = append(resourceVersions, configResourceVersions...)
+
 	cliConfig, _, err := resourceapply.ApplyConfigMap(c.configMaps, c.recorder, expectedCLIconfig)
 	if err != nil {
 		return err
 	}
+	resourceVersions = append(resourceVersions, cliConfig.GetResourceVersion())
 
 	// deployment, have RV of all resources
 	// TODO use ExpectedDeploymentGeneration func
-	// TODO manually get RV of all the config maps and secrets in syncData
 	// TODO we also need the RV for the serving-cert secret (servingCertName)
```

Comment:
Isn't the CA bundle enough?

Reply:
The serving-cert could be rotated independently of the CA.